test/setup.ts

```diff
@@ -408,3 +408,10 @@ global.requestAnimationFrame = (callback) => {
 };
 
 global.performance.now = () => Date.now();
+
+console.error = () => {
+    // empty
+};
+console.warn = () => {
+    // empty
+};
```

Comment:
Instead of suppressing error and warn logs, we can filter out only those known or safe to ignore. There are cases where these logs are important to be seen so that any incorrect handling/setup of tests, missing props/mocks etc. are addressed timely. In some cases, logs are helpful to identify why some tests are taking longer than usual. In webapp, it's done as: https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/tests/setup_jest.ts#L72-L103

Reply:
I'll change it back.. all works well without this just wanting to remove some noise and definitely does not need what was done on webapp